cloudbuild.yaml

```diff
@@ -4,7 +4,7 @@ timeout: 3600s
 options:
   substitution_option: ALLOW_LOOSE
 steps:
-  - name: 'golang:1.20.10'
+  - name: 'golang:1.20.12'
```

Comment:
Also unclear to me if this should be changed, as I don't see it updated in, e.g., #595 or #604.

Reply:
Oversight, I think it should have been updated.